### ISSUE TITLE
g_cor term computation

### ISSUE THREAD

**Issue by takaaki-sanjoh** [nicola-decao/s-vae-pytorch#7]
Hi,
I have read your paper and it's very interesting.
Now, I'm trying to comprehend your MNIST example implementation.

I found this program take kl_divergence and BCEWithLogitsLoss in loss function.
So I think this program compute a gradient of KL divergence and g_rep term through backpropagation, but not g_cor term.

Why don't you put (log p(x|z) * log g/r) term into loss function for g_cor calculation?
Or can I ask you where g_cor term is computed?

**Comment by trdavidson** [comment]
Hi exp-3,

Thank you for your interest in our work! In our initial experiments, we wrote an implementation of the `g_cor` term, but found that in practice it did not significantly effect results.

In fact, as noted in [Naesseth et al. (2017)](https://arxiv.org/pdf/1610.05683.pdf), the 'gradient correction' term, can often be omitted. 

If you do wish to implement this yourself,  you could take a look at [Appendix D.2, equation](https://arxiv.org/pdf/1804.00891.pdf) of our work to see the explicit form written out.

Good luck!
